packages/@webex/plugin-presence/src/presence.ts

```diff
@@ -46,6 +46,20 @@ const Presence: IPresence = WebexPlugin.extend({
     });
   },
 
+  /**
+   * Initializes the presence worker.
+   * @returns {undefined}
+   */
+  initializeWorker(): void {
+    if (this.webex.ready) {
+      this.worker.initialize(this.webex);
+    } else {
+      this.webex.once('ready', () => {
+        this.worker.initialize(this.webex);
+      });
+    }
+  },
```

Comment:
Making the same changes here, even though cantina does not use the external presence plugin, just to keep them in sync.

Reply:
Thanks, Charles. Good call